test/src/main/java/org/apache/accumulo/test/manager/SuspendedTabletsIT.java

```diff
@@ -140,6 +193,7 @@ public void shutdownAndResumeTserver() throws Exception {
       }
 
       // remove servers with metadata on them from the list of servers to be shutdown
+      assertEquals("Expecting a single tServer in metadataServerSet", 1, metadataServerSet.size());
       tserverSet.removeAll(metadataServerSet);
 
       assertEquals("Expecting two tServers in shutdown-list", 2, tserverSet.size());
```

Comment:
```suggestion
      assertEquals("Expecting " + (TSERVERS - 1) + " tServers in shutdown-list", TSERVERS - 1, tserverSet.size());
```

Reply:
This isn't a line I changed, but I did notice that the value 2 is hard-coded a bunch of places in the test. If I'm going to fix one, then they should probably all be fixed.